server/bin/dev.js

```diff
@@ -30,7 +33,7 @@ module.exports = function(app, devServer) {
     app.get('/', android);
     app.get('/legal', android);
     app.get(`/share/:id${ID_REGEX}`, android);
-    app.get(`/download/:id${ID_REGEX}`, android);
+    app.get(`/download/:id${ID_REGEX}/:key`, android);
```

Comment:
this needs to go on the choo route in `android.js` not the express route

Reply:
Ok sure that makes sense... But this fixes the bug???

Reply:
nevermind, i was confused about what we're trying to do here